tests/dummy/app/application/template.hbs

```diff
@@ -1,3 +1,3 @@
-{{#nrg-application title="NRG UI"}}
+{{#nrg-application title="NRG UI" clicked=(route-action 'clickedLink')}}
```

Comment:
please run `npm run lint:hbs` and resolve any issues.

Reply:
I ran it, but it did not report any issues. The single quotes for this code was changed to double quotes in my last commit.